Content.Server/_CorvaxNext/BodyCam/BodyCameraComponent.cs

```diff
@@ -0,0 +1,12 @@
+using Robust.Shared.GameObjects;
+
+namespace Content.Server.Camera
+{
+    /// <summary>
+    /// A marker component for cameras that should only be active when worn.
+    /// </summary>
+    [RegisterComponent]
+    public sealed partial class BodyCameraComponent : Component
+    {
+    }
```

Comment:
```suggestion
    public sealed partial class BodyCameraComponent : Component;
```

Почини свой IDE и делай так.